software/base/src/main/java/brooklyn/entity/software/MachineLifecycleEffectorTasks.java

```diff
@@ -561,7 +560,7 @@ public void stop(ConfigBag parameters) {
 
         boolean isEntityStopped = entity().getAttribute(SoftwareProcess.SERVICE_STATE_ACTUAL)==Lifecycle.STOPPED;
 
-        DynamicTasks.queue("pre-stop", new Callable<String>() { public String call() {
+        DynamicTasks.queue("pre-stop of " + entity() + ", submitted by " + Tasks.current(), new Callable<String>() { public String call() {
```

Comment:
entity and submitted are available as metadata on the task already. generally i think it works better in the gui if the task name is short. is there a good reason to include it in the name?